language-support/java/bindings-rxjava/src/main/java/com/daml/ledger/rxjava/grpc/CommandClientConfig.java

```diff
@@ -0,0 +1,238 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.rxjava.grpc;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
+import static java.util.Optional.empty;
+
+import com.daml.ledger.javaapi.data.codegen.HasCommands;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.List;
+import java.util.Optional;
+import org.checkerframework.checker.nullness.qual.NonNull;
+
+/**
+ * TODO: javadoc this
+ * This class represents a set of parameters required for {@link com.daml.ledger.rxjava.CommandClient}
```

Comment:
This is not really meant to configure the client itself, bur rather to build a valid submission. I would probably consider renaming it to `CommandsBuilder` or something along those lines. In any case, I would not drive the user into thinking that this configures the client itself.

Reply:
Doesn't `CommandsBuilder` indicate it's a builder for commands?  

How about `CommandsSubmissionParams`?

Reply:
I read the API a bit more and I see why `CommandsBuilder` makes sense now. Refactored

Reply:
Please rename to `CommandsSubmission`